test/test_packages/RuntestsArgs/test/runtests.jl

```diff
@@ -0,0 +1,6 @@
+using RuntestsArgs
+using Test
+
+@testset "RuntestsArgs.jl" begin
+    @test all(ARGS .== RuntestsArgs.expected_arguments())
+end
```

Comment:
I would also add tests for julia_args, e.g. by inspecting `Base.JLOptions()` for some values that you set from `Pkg.test(; julia_args = ...)`

Reply:
Done.